web/viewer.js

```diff
@@ -901,6 +902,12 @@ var PDFViewerApplication = {
                      store.get('zoom', self.pdfViewer.currentScale);
           var left = store.get('scrollLeft', '0');
           var top = store.get('scrollTop', '0');
+          rotation = store.get('rotation', '0');
+
+          self.setScale(DEFAULT_SCALE, true);
```

Comment:
I am now setting the scale and moved the code back to its original spot. There is no error. Can this be considered a viable solution?